.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,5 @@
+eclipse.preferences.version=1
+encoding//src/main/java=UTF-8
+encoding//src/main/resources=UTF-8
+encoding//src/test/java=UTF-8
+encoding/<project>=UTF-8
```

Comment:
This is an IDE-specific folder (eclipse-specific, to be exact) that needs not be committed to the repo. You should add it to `.gitignore`.